tests/odbc/test-odbc-oracle.cpp

```diff
@@ -0,0 +1,301 @@
+//
+// Copyright (C) 2004-2006 Maciej Sobczak, Stephen Hutton, David Courtney
+// Distributed under the Boost Software License, Version 1.0.
+// (See accompanying file LICENSE_1_0.txt or copy at
+// http://www.boost.org/LICENSE_1_0.txt)
+//
+
+#include "soci/soci.h"
+#include "soci/odbc/soci-odbc.h"
+#include "common-tests.h"
+#include <iostream>
+#include <string>
+#include <cstdio>
+#include <ctime>
+#include <cmath>
+
+using namespace soci;
+using namespace soci::tests;
+
+// A generic version class: we might want to factor it out later if it is
+// needed elsewhere (it would probably also need to be renamed to something
+// less generic then).
+class odbc_version
```

Comment:
This doesn't seem needed at all here, PostgreSQL test actually uses the version, but this one doesn't.